File: Programmers/Lv2/최댓값과 최솟값.py
def solution(s):
    list_s = s.split(' ')
    cvt = []
    for w in list_s:
        cvt.append(int(w))

    cvt.sort()
    answer = ''
    answer += str(cvt[0])
    answer += ' '
    answer += str(cvt[-1])
    return answer

File: Programmers/Lv2/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <sstream>

using namespace std;

string solution(string s) {
    stringstream ss(s);
    string tmp;
    
    vector<int> v;
    
    while(ss >> tmp){
        v.push_back(stoi(tmp));    
    }
    
    int max = *max_element(v.begin(), v.end());
    int min = *min_element(v.begin(), v.end());
    
    string answer = to_string(min) + " " + to_string(max);
    return answer;
}